<?xml version="1.0" encoding="UTF-8"?>
<testng-results total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-02-23T17:33:45Z" name="Suite" finished-at="2021-02-23T17:34:00Z" duration-ms="14920">
    <groups>
    </groups>
    <test started-at="2021-02-23T17:33:45Z" name="Test" finished-at="2021-02-23T17:34:00Z" duration-ms="14920">
      <class name="AutomationFramework.Homepage">
        <test-method is-config="true" signature="initialize()[pri:0, instance:AutomationFramework.Homepage@61078690]" started-at="2021-02-23T11:33:45Z" name="initialize" finished-at="2021-02-23T11:33:57Z" duration-ms="11943" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialize -->
        <test-method signature="homepagaeNavigation(java.lang.String, java.lang.String)[pri:0, instance:AutomationFramework.Homepage@61078690]" started-at="2021-02-23T11:33:57Z" name="homepagaeNavigation" data-provider="getdata" finished-at="2021-02-23T11:34:00Z" duration-ms="2912" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[game119@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[test@123]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [MY ACCOUNT] but found [AUTHENTICATION]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [MY ACCOUNT] but found [AUTHENTICATION]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:176)
	at org.testng.Assert.assertEquals(Assert.java:186)
	at AutomationFramework.Homepage.homepagaeNavigation(Homepage.java:46)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:386)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:323)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:143)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- homepagaeNavigation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:AutomationFramework.Homepage@61078690]" started-at="2021-02-23T11:34:00Z" name="tearDown" finished-at="2021-02-23T11:34:00Z" duration-ms="51" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- AutomationFramework.Homepage -->
      <class name="AutomationFramework.AppTest">
        <test-method signature="shouldAnswerWithTrue()[pri:0, instance:AutomationFramework.AppTest@332a7fce]" started-at="2021-02-23T11:34:00Z" name="shouldAnswerWithTrue" finished-at="2021-02-23T11:34:00Z" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- shouldAnswerWithTrue -->
      </class> <!-- AutomationFramework.AppTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
